tests/test_integration_fairflow.py

```diff
@@ -3,20 +3,14 @@
 """
 
 import json
-import time
-import requests
-import pytest
-import openreview
-import logging
-import datetime
 
-from matcher.service.openreview_interface import ConfigNoteInterface
-from matcher.solvers import SolverException
+import openreview
+import pytest
 
 from conftest import clean_start_conference, wait_for_status
 
 
-def test_integration_basic(openreview_context):
+def test_integration_basic(openreview_context, celery_app, celery_worker):
```

Comment:
why do you define these two parameter in every test function? I don't see they are being used.

Reply:
They are the pytest fixtures provided by celery to replicate the celery_app and celery_worker. `celery_worker` starts a worker in separate thread and terminates it at the end of the test. So we don't have to manually start a worker. I think we might not need the celery_app fixture. I'll check that.